tests/validation/tests/v3_api/test_v2_cis_scan.py

```diff
@@ -0,0 +1,158 @@
+
+    # check if CIS is already installed
+    app = client.list_catalog_cattle_io_app()
+    for app in app["data"]:
+        if app["metadata"]["name"] == "rancher-cis-benchmark":
+            response = client.action(obj=app, action_name="uninstall")
+            app_list = wait_until_app_v2_uninstall(
+                client,
+                "rancher-cis-benchmark"
+            )
+            assert "rancher-cis-benchmark" not in app_list, \
+                "App has not uninstalled"
+        if app["metadata"]["name"] == "rancher-cis-benchmark-crd":
+            response = client.action(obj=app, action_name="uninstall")
+            app_list = wait_until_app_v2_uninstall(
+                client,
+                "rancher-cis-benchmark-crd"
+            )
+            assert "rancher-cis-benchmark-crd" not in app_list, \
+                "App CRD has not uninstalled"
+
+    # create namespace
+    command = "create namespace cis-operator-system"
+    execute_kubectl_cmd(command, False)
+
+    # install CIS v2
+    cis_charts["annotations"] = cis_annotations
+    cis_charts["values"]["global"]["cattle"]["clusterId"] = cluster_id
+    cis_charts["values"]["global"]["cattle"]["clusterName"] = cluster_name
+    cis_charts["chartName"] = "rancher-cis-benchmark-crd"
+    cis_charts["releaseName"] = "rancher-cis-benchmark-crd"
+
+    # install CRD
+    response = client.action(obj=rancher_repo, action_name="install",
+                             charts=[cis_charts],
+                             namespace="cis-operator-system",
+                             disableOpenAPIValidation=False,
+                             noHooks=False,
+                             projectId=None,
+                             skipCRDs=False,
+                             timeout="600s",
+                             wait=True)
+    print("response", response)
+    app_list = wait_until_app_v2_deployed(client, "rancher-cis-benchmark-crd")
+    assert "rancher-cis-benchmark-crd" in app_list
+
+    # install app
+    cis_charts["chartName"] = "rancher-cis-benchmark"
+    cis_charts["releaseName"] = "rancher-cis-benchmark"
+    response = client.action(obj=rancher_repo,action_name="install",
+                        charts=[cis_charts],
+                        namespace="cis-operator-system",
+                        disableOpenAPIValidation=False,
+                        noHooks=False,
+                        projectId=None,
+                        skipCRDs=False,
+                        timeout="600s",
+                        wait=True)
+    print("response",response)
+    app_list = wait_until_app_v2_deployed(client, "rancher-cis-benchmark")
+    assert "rancher-cis-benchmark" in app_list
+
+
+@pytest.fixture(scope='module', autouse="True")
+def create_project_client(request):
+    client, cluster_detail["cluster"] = get_user_client_and_cluster()
+
+
+def wait_until_app_v2_deployed(client, app_name, timeout=DEFAULT_APP_V2_TIMEOUT):
```

Comment:
Could these `v2_deployed` and `v2_uninstall` be shared among all charts? If so it would be a good idea to shared them in a common lib, `common.py` is getting very convoluted maybe we can have a `common_v2` or something similar?

Reply:
Addressed this in the PR. I have added them in common.py for now.